src/components/src/main/java/org/apache/jmeter/visualizers/backend/SamplerMetric.java

```diff
@@ -99,8 +99,9 @@ public static void setDefaultWindowMode(WindowMode windowMode) {
     /**
      * Add a {@link SampleResult} to be used in the statistics
      * @param result {@link SampleResult} to be used
+     * @param isCumulated is the overall Sampler Metric
      */
-    public synchronized void add(SampleResult result) {
+    public synchronized void add(SampleResult result, boolean isCumulated) {
```

Comment:
Could we use two separate methods instead of a boolean flag?

I think

`samplerMetric.add(sampleResult, false)`
`getSamplerMetric(CUMULATED_METRICS).add(sampleResult, true);`

isn't as readable as:

`samplerMetric.add(sampleResult)`
`getSamplerMetric(CUMULATED_METRICS).addCumulative(sampleResult);`